python-lib/config.py

```diff
@@ -82,27 +82,38 @@ def get_hdfs_location(dataset_config: Mapping[AnyStr, Any], sf_stage_name: AnySt
     return f'{sf_stage_name}{path}/'
 
 
-def get_snowflake_to_hdfs_query(sf_location: AnyStr, sf_table_name: AnyStr) -> AnyStr:
+def get_snowflake_to_hdfs_query(sf_location: AnyStr, sf_table_name: AnyStr,
+                                sf_schema: List[Mapping[AnyStr, AnyStr]]) -> AnyStr:
```

Comment:
The parameter you are passing in does not match this type definition. Instead of updating the type definition to reflect the parameter you're passing, you should transform `['schema']['columns']` in to column to type mappings outside of this function—put it in the call from the recipe.

![image](https://user-images.githubusercontent.com/939816/87525088-ed71e780-c680-11ea-810d-5fe1ca6bd3d4.png)

Reply:
🤦‍♂️